<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvaliacaoMateriasDialog</class>
 <widget class="QDialog" name="AvaliacaoMateriasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Avaliação de Dificuldade</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* ====== FUNDO PRINCIPAL ====== */
QDialog#AvaliacaoMateriasDialog {
    background-image: url(:/images/imagens/fundo_2.png);
    background-position: center;
    background-repeat: no-repeat;
}

/* ====== BARRA DE NAVEGAÇÃO ====== */
QLabel#navBar {
    background-color: #0f1011;
}

/* ====== WIDGET CENTRAL ====== */
QWidget#centralWidget {
    background-color: rgba(26, 22, 26, 0.85);
    border-radius: 15px;
}

/* ====== TÍTULO ====== */
QLabel#tituloLabel {
    color: #FFFFFF;
    font-size: 26px;
    font-weight: bold;
    background-color: transparent;
}

/* ====== SUBTÍTULO ====== */
QLabel#subtituloLabel {
    color: #D3AF35;
    font-size: 13px;
    background-color: transparent;
}

/* ====== COMBOBOX ====== */
QComboBox {
    background-color: rgba(66, 55, 56, 0.9);
    color: #F4B315;
    border: 2px solid #F4B315;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 14px;
    font-weight: bold;
}

QComboBox:hover {
    border-color: #D3AF35;
    background-color: rgba(82, 68, 71, 0.9);
}

QComboBox::drop-down {
    border: none;
    width: 30px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 6px solid #F4B315;
    margin-right: 8px;
}

QComboBox QAbstractItemView {
    background-color: #423738;
    color: #F4B315;
    selection-background-color: #F4B315;
    selection-color: #1A161A;
    border: 2px solid #F4B315;
}

/* ====== SCROLL AREA ====== */
QScrollArea {
    border: none;
    background-color: transparent;
}

QScrollBar:vertical {
    background-color: #423738;
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background-color: #F4B315;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background-color: #D3AF35;
}

QWidget#containerMaterias {
    background-color: transparent;
}

/* ====== LOGO E ÍCONES ====== */
QLabel#logoLabel, QLabel#homeButton, QLabel#perfilButton {
    background-color: transparent;
    border-radius: 50px;
}
   </string>
  </property>
  
  <layout class="QVBoxLayout" name="mainLayoutOuter">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   
   <item>
    <!-- BARRA DE NAVEGAÇÃO SUPERIOR -->
    <widget class="QWidget" name="navWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     
     <!-- Logo UTFPR -->
     <widget class="QLabel" name="logoLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/images/imagens/Logo_1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     
     <!-- Botão Home -->
     <widget class="QLabel" name="homeButton">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>20</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/images/imagens/inicio.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     
     <!-- Botão Perfil -->
     <widget class="QLabel" name="perfilButton">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>20</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/images/imagens/perfil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     
     <!-- Fundo da barra -->
     <widget class="QLabel" name="navBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0f1011;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     
     <zorder>navBar</zorder>
     <zorder>logoLabel</zorder>
     <zorder>homeButton</zorder>
     <zorder>perfilButton</zorder>
    </widget>
   </item>
   
   <item>
    <!-- Widget central -->
    <widget class="QWidget" name="centralWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      
      <!-- CABEÇALHO -->
      <item>
       <layout class="QVBoxLayout" name="headerLayout">
        <item>
         <widget class="QLabel" name="tituloLabel">
          <property name="text">
           <string>Avaliação de Dificuldade das Matérias</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        
        <item>
         <widget class="QLabel" name="subtituloLabel">
          <property name="text">
           <string>Veja a opinião dos alunos sobre cada matéria e contribua com sua avaliação</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      
      <!-- LINHA SEPARADORA -->
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">background-color: #F4B315;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      
      <!-- FILTRO POR CATEGORIA -->
      <item>
       <layout class="QHBoxLayout" name="filtroLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filtrar por categoria:</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #F4B315; font-size: 14px; font-weight: bold; background: transparent;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categoriaComboBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      
      <!-- ÁREA DE CONTEÚDO -->
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <widget class="QWidget" name="containerMaterias">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1126</width>
           <height>530</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 
 <resources>
  <include location="icons.qrc"/>
 </resources>
 
 <connections/>
</ui>